core/src/ast/types.rs

```diff
@@ -53,6 +53,14 @@ impl CustomType {
         }
     }
 
+    /// Get all rust links on this type and its methods
+    pub fn all_rust_links(&self) -> impl Iterator<Item = &RustLink> + '_ {
+        [self.docs()]
+            .into_iter()
```

Comment:
Nit (very optional): `core::iter::once(self.docs())` instead of `[self.docs()].into_iter()`